terraform/modules/data-share-service/ecs-security.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_security_group" "ecs_tasks" {
+  name_prefix = "${var.environment}-ecs-tasks-"
+  description = "For GDX Data Share PoC Service ECS tasks, inbound access from GDX LB only"
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    protocol        = "tcp"
+    from_port       = var.service_port
+    to_port         = var.service_port
+    security_groups = aws_security_group.lb_auto.*.id
+  }
+
+  egress {
```

Comment:
This is a broad "allow" right? I think that's fine for POC, but we should lock this down as and when things have stabilised and we know what legit outbound traffic is.

Reply:
Other services I have seen have been perfectly happy with this, and it is the default with AWS, but am happy to lock down later